site/content/en/docs/Advanced/high-availability-agones.md

```diff
@@ -34,19 +38,40 @@ An important configuration to note is the PodDisruptionBudget fields, `agones.ex
 
 ## Deployment Considerations
 
-When `SplitControllerAndExtensions` is enabled, what was previously a single `agones-controller` pod is deployed as `agones-controller` and 2 `agones-extensions` pods. For example: 
+{{% feature publishVersion="1.31.0" %}}
+Leader election will automatically be enabled while `SplitControllerAndExtensions` is enabled and while `agones.controller.replicas` is set to > 1. The current default value for [`agones.controller.replicas`]({{< relref "/docs/Installation/Install Agones/helm.md#configuration" >}}) is 2.
```

Comment:
* Speak in present, since this is docs: _Leader election *is* automatically enabled when_ (also I think `when` makes more sense than `while` here).
* I think you can make this a bit more concise and say: "Leader election is automatically enabled when `SplitControllerAndExtensions` is enabled, and [`agones.controller.replicas`](relref) is > 1. `agones.controller.replicas` defaults to 2.